modules/programs/tmux.nix

```diff
@@ -41,6 +41,7 @@ in
 {
   imports = [
     (mkRenamedOptionModule [ "programs" "tmux" "tmuxConfig" ] [ "programs" "tmux" "extraConfig" ])
+    (mkRemovedOptionModule [ "programs" "tmux" "defaultCommand" ] "Use `programs.tmux.extraConfig` to configure the default command instead. If unset, tmux will default to using your system configured login shell.")
```

Comment:
Is there a reason to get rid of this?

Reply:
I couldn't find any examples of anyone using this online and it seemed easy enough to configure without this option:

```nix
programs.tmux.extraConfig = ''
  set -g default-command "zsh -l"
'';
```

So I decided it was easier to just remove this option than add support for a `null` default (as we no longer have `environment.loginShell`)

Reply:
Happy to add it back in if you prefer